kotlintest-tests/kotlintest-tests-core/src/test/kotlin/com/sksamuel/kotlintest/runner/junit5/JUnitTestRunnerListenerTest.kt

```diff
@@ -32,8 +32,8 @@ class JUnitTestRunnerListenerTest : WordSpec({
       val listener = JUnitTestRunnerListener(mock, rootDescriptor)
 
       val spec = JUnitTestRunnerListenerTest()
-      listener.prepareSpec(spec.description(), spec::class)
-      listener.completeSpec(spec.description(), spec::class, null)
+      listener.beforeSpecClass(spec.description(), spec::class)
```

Comment:
If we do things like this, we're not guaranteeing that our deprecated methods work anymore. Is that ok?